src/views/box_view.rs

```diff
@@ -263,4 +265,25 @@ mod tests {
         assert_eq!(Vec2::new(10, 1), full.required_size(Vec2::new(10, 1)));
         assert_eq!(Vec2::new(10, 10), full.required_size(Vec2::new(10, 10)));
     }
+
+    #[test]
+    fn test_get_inner() {
```

Comment:
I'm not sure this tests should stay.
I can remove it or change.

Reply:
Tests are never bad!